pages/club/[club]/index.js

```diff
@@ -1,12 +1,12 @@
 import { useState, useEffect } from 'react'
 import s from 'styled-components'
 
-import renderPage from '../renderPage.js'
-import { doApiRequest } from '../utils'
-import Tabs from '../components/ClubPage/Tabs'
-import Header from '../components/ClubPage/Header'
-import InfoBox from '../components/ClubPage/InfoBox'
-import SocialIcons from '../components/ClubPage/SocialIcons'
+import renderPage from '../../../renderPage.js'
```

Comment:
Why have `.js` on the end of this?

Reply:
Fixed, note that these changes were introduced in #59 and will also appear in that merge request.